src/main/java/frc/robot/RobotContainer.java

```diff
@@ -88,7 +88,11 @@ public class RobotContainer implements Logged {
   int[] rotationColor;
   int[] XTranslationColor;
   int[] YTranslationColor;
-
+  int[] previousRotationColor;
+  int[] previousXTranslationColor;
+  int[] previousYTranslationColor;
+  int[] previousRotationCorrect;
```

Comment:
You'd want a boolean, not an int array, since this is just checking if the previous rotation is correct or not